warehouse/query-core/src/main/java/datawave/query/jexl/functions/GeoWaveFunctionsDescriptor.java

```diff
@@ -53,7 +68,9 @@ public GeoWaveJexlArgumentDescriptor(ASTFunctionNode node, String name, List<Jex
         public JexlNode getIndexQuery(ShardQueryConfiguration config, MetadataHelper helper, DateIndexHelper dateIndexHelper, Set<String> datatypeFilter) {
```

Comment:
@ivakegg I'm going to update this tomorrow morning to pull the max expansion/envelopes from the configuration here!  Stay tuned!

Reply:
@ivakegg Added config params to the shard query configuration...